tests/ipex/test_modeling.py

```diff
@@ -255,6 +256,20 @@ def test_pipeline(self, model_arch):
         self.assertEqual(pipe.device, model.device)
         self.assertTrue(all("This is a sample" in item["generated_text"] for item in outputs))
 
+    @parameterized.expand(SUPPORTED_ARCHITECTURES)
```

Comment:
can you test different beam search / use_cache to True/False and compare the generated outputs with the ones from the original transformers model, also for couple of generation config (beam search but also greedy), this can be done for `IPEX_PATCHED_SUPPORTED_ARCHITECTURES = ("llama",)` as not other architectures currently support the ipex model patching